go.mod

```diff
@@ -51,17 +51,17 @@ require (
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 
-	//pinned k8s.io to v0.26.x tag
-	k8s.io/api v0.26.13
-	k8s.io/apiextensions-apiserver v0.26.13
-	k8s.io/apimachinery v0.26.13
-	k8s.io/cli-runtime v0.26.13
-	k8s.io/client-go v0.26.13
-	k8s.io/code-generator v0.26.13
-	k8s.io/kube-openapi v0.0.0-20230109183929-3758b55a6596
-	k8s.io/kubectl v0.26.13
-	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
-	sigs.k8s.io/controller-runtime v0.14.7
+	//pinned k8s.io to v0.29.x tag
+	k8s.io/api v0.29.0
+	k8s.io/apiextensions-apiserver v0.29.0
+	k8s.io/apimachinery v0.29.0
+	k8s.io/cli-runtime v0.29.0
+	k8s.io/client-go v0.29.0
+	k8s.io/code-generator v0.29.0
+	k8s.io/kubectl v0.29.0
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
+	sigs.k8s.io/controller-runtime v0.16.3
+	sigs.k8s.io/kustomize/kyaml v0.14.3-0.20230601165947-6ce0bf390ce3 // indirect
```

Comment:
Is kustomize pinned version or indirect? If its not pinned, lets move it to indirect dependencies over [here](https://github.com/kanisterio/kanister/pull/2532/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R69)

Reply:
Done